frontend/src/metabase/dashboard/components/DashCard/DashCardActionsPanel/DashCardActionsPanel.tsx

```diff
@@ -152,7 +153,11 @@ function DashCardActionsPanelInner({
   }
 
   if (!isLoading && !hasError) {
-    if (dashcard && !isVirtualDashCard(dashcard)) {
+    if (
+      dashcard &&
+      !isVirtualDashCard(dashcard) &&
+      dashboardCardSupportsVisualizer(dashcard)
+    ) {
```

Comment:
`isVirtualDashCard` shouldn't be needed here anymore since `dashboardCardSupportsVisualizer` checks for specific display types and virtual cards can't have them